packages/server/tests/account.test.ts

```diff
@@ -33,6 +36,24 @@ describe('auth', () => {
     expect(typeof res.data.jwt).toBe('string')
   })
 
+  it('fails on invalid usernames', async () => {
+    try {
+      await client.todo.adx.createAccount(
+        {},
+        {
+          username: 'did:bad-username.test',
+          did: 'bad.test',
+          password: 'asdf',
+        },
+      )
+      throw new Error('Didnt throw')
+    } catch (e: any) {
+      expect(
+        e instanceof TodoAdxCreateAccount.InvalidUsernameError,
```

Comment:
hell yea

Reply:
We certainly _could_ get the error types in the server lexicon, but what I did instead was type the response so that `error` has to be one of the declared error names. That means you'll get a type error on invalid ones, and the autocomplete will help you out. I figured that was simpler than having to import the error classes, but we can do both.

Reply:
ah got it. think string types are fine for now 👍